packages/core/src/App/Containers/RealAccountSignup/address-details.jsx

```diff
@@ -37,206 +37,170 @@ const InputField = props => {
-    }
-
-    handleValidate = values => {
-        const { errors } = splitValidationResultTypes(this.props.validate(values));
+    const handleValidate = values => {
+        const { errors } = splitValidationResultTypes(props.validate(values));
         return errors;
     };
 
-    render() {
-        return (
-            <Formik
-                initialValues={this.props.value}
-                validate={this.handleValidate}
-                validateOnMount
-                onSubmit={(values, actions) => {
-                    if (values.address_state && this.props.states_list.length) {
-                        values.address_state = this.state.address_state_to_display
-                            ? getLocation(this.props.states_list, this.state.address_state_to_display, 'value')
-                            : getLocation(this.props.states_list, values.address_state, 'value');
-                    }
-                    this.props.onSubmit(
-                        this.props.getCurrentStep() - 1,
-                        values,
-                        actions.setSubmitting,
-                        this.props.goToNextStep
-                    );
-                }}
-            >
-                {({ handleSubmit, isSubmitting, errors, values, setFieldValue }) => (
-                    <AutoHeightWrapper default_height={350} height_offset={isDesktop() ? 80 : null}>
-                        {({ setRef, height }) => (
-                            <form ref={setRef} onSubmit={handleSubmit}>
-                                <Div100vhContainer
-                                    className='details-form'
-                                    height_offset='110px'
-                                    is_disabled={isDesktop()}
+    return (
+        <Formik
+            initialValues={props.value}
+            validate={handleValidate}
+            validateOnMount
+            onSubmit={(values, actions) => {
+                if (values.address_state && props.states_list.length) {
+                    values.address_state = address_state_to_display
+                        ? getLocation(props.states_list, address_state_to_display, 'value')
+                        : getLocation(props.states_list, values.address_state, 'value');
+                }
+                props.onSubmit(props.getCurrentStep() - 1, values, actions.setSubmitting, props.goToNextStep);
+            }}
+        >
+            {({ handleSubmit, isSubmitting, errors, values, setFieldValue }) => (
+                <AutoHeightWrapper default_height={350} height_offset={isDesktop() ? 80 : null}>
+                    {({ setRef, height }) => (
+                        <form ref={setRef} onSubmit={handleSubmit}>
+                            <Div100vhContainer className='details-form' height_offset='110px' is_disabled={isDesktop()}>
+                                <p className='details-form__description'>
```

Comment:
Is using `Text` going to be done in a separate card?

Reply:
I think it's best if we leave it to @sara-fs to help out on that

Reply:
@sara-fs Add this to your list if it's not already :tada: :smile: